testsuite/integration-arquillian/servers/auth-server/services/testsuite-providers/src/main/java/org/keycloak/testsuite/rest/TestingResourceProvider.java

```diff
@@ -224,7 +226,7 @@ public Response revertTestingInfinispanTimeService() {
     @GET
     @Path("/get-client-sessions-count")
     @Produces(MediaType.APPLICATION_JSON)
-    public Integer getClientSessionsCountInUserSession(@QueryParam("realm") final String name, @QueryParam("session") final String sessionId) {
+    public Integer getClientSessionsCountInssion(@QueryParam("realm") final String name, @QueryParam("session") final String sessionId) {
```

Comment:
Minor: Not sure why this method was updated? New name does not makes much sense :-) (even if it probably does not break anything...). I guess it was a typo?

Reply:
Not sure how that happend, was a mistake. Fixed in [a65798d](https://github.com/keycloak/keycloak/pull/27931/commits/a65798dfaa126bc4edbcc254a77affa5c24816bc)